shared/AppInsightsCommon/src/Utils/TraceParent.ts

```diff
@@ -17,32 +17,6 @@ export const INVALID_TRACE_ID = "00000000000000000000000000000000";
 export const INVALID_SPAN_ID = "0000000000000000";
 const SAMPLED_FLAG = 0x01;
 
-/**
```

Comment:
Just trying to understand why this function is being moved and made public. Is it no longer intended to just be a util function? Or just getting moved out of core for dependency reasons?

Reply:
This was duplicated code, it was already there in different utils file